pom.xml

```diff
@@ -15,7 +15,7 @@
 
   <groupId>com.spotify.confidence</groupId>
   <artifactId>openfeature-provider</artifactId>
-  <version>0.0.4-SNAPSHOT</version>
+  <version>${project.version}</version>
```

Comment:
Will this just work* with release please?

Reply:
This also make mvn spit a new warning:
```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for com.spotify.confidence:openfeature-provider:jar:0.0.4-SNAPSHOT
[WARNING] 'version' contains an expression but should be a constant. @ com.spotify.confidence:openfeature-provider:${project.version}, /Users/fdema/Developer/github/confidence-openfeature-provider-java/pom.xml, line 18, column 12
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```